FBSnapshotTestCase/FBSnapshotTestCasePlatform.m

```diff
@@ -34,7 +34,7 @@ BOOL FBSnapshotTestCaseIs64Bit(void)
 NSString *FBDeviceAgnosticNormalizedFileName(NSString *fileName)
 {
   UIDevice *device = [UIDevice currentDevice];
-  CGSize screenSize = [[UIApplication sharedApplication] keyWindow].bounds.size;
```

Comment:
I had similar problem:
I write some framework which is using UIApplication. Same framework doesn't have UIApplication, so I created TestApp, which hosts tests.
So check if this resolve your problem:
1. Add new Application target
2. Choose your framework test target
3. Go to `general` tab
4. Set your new Application target as `Host Application`
5. Make sure that your Application target contains framework tests in test schema